@here/harp-mapview/lib/ScreenProjector.ts

```diff
@@ -92,6 +92,37 @@ export class ScreenProjector {
         return undefined;
     }
 
+    /**
+     * Test if the area around the specified point is visible on the screen.
+     *
+     * @param {(Vector3Like)} source The centered source vector to project.
+     * @param {(Number)} width Half of the width of the area in NDC space [0..1].
+     * @param {(Number)} height Half of the height of the area in NDC space [0..1].
+     * @param {THREE.Vector2} target The target vector.
+     * @returns {THREE.Vector2} The projected vector (the parameter 'target') or undefined if
+     * the area is completely outside the screen.
+     */
+    projectAreaToScreen(
+        source: Vector3Like,
+        width: number,
+        height: number,
```

Comment:
better call them halfWidth and halfHeight

Reply:
will do